### DISCUSSION TITLE
Getting lists with length over 1100 when using GetLidarObstacles()

### DISCUSSION THREAD

**Discussion by jbain1029** [SimDynamX/SpaceTeamsPRO#68] [2024 Lunar SAR Competition Q&A] [answered]
When I use the GetLidarObstacles() command to determine the location of potential obstacles, it returns over 1100 vectors. Is this supposed to happen? Also, all of the radii are the same even when the LTV is right next to an obstacle

**Comment by connorjak** [comment]
That seems like a bug; please post your log files. Also, maybe your repo doesn't match the v1.2.0 STU_Lunar_SAR code (should 100% match with the exception of MissionManager and EntityBehavior python files, and changes to the Task1/Task2 sim configs to adjust the Source paths to match your repo's names)

**Reply by jbain1029** [reply]
[lidar_data.csv](https://github.com/user-attachments/files/17724043/lidar_data.csv)

[Sim_STU2_SAR_Task2_PySys_CompetitionBackend.log](https://github.com/user-attachments/files/17724046/Sim_STU2_SAR_Task2_PySys_CompetitionBackend.log)
[Sim_STU2_SAR_Task2_PySys_EntityBehavior_LTV1.log](https://github.com/user-attachments/files/17724047/Sim_STU2_SAR_Task2_PySys_EntityBehavior_LTV1.log)
[Sim_STU2_SAR_Task2_PySys_EntityBehavior_LTV2.log](https://github.com/user-attachments/files/17724048/Sim_STU2_SAR_Task2_PySys_EntityBehavior_LTV2.log)
[Sim_STU2_SAR_Task2_PySys_EntityBehavior_Scout1.log](https://github.com/user-attachments/files/17724049/Sim_STU2_SAR_Task2_PySys_EntityBehavior_Scout1.log)
[Sim_STU2_SAR_Task2_PySys_EntityBehavior_Scout2.log](https://github.com/user-attachments/files/17724050/Sim_STU2_SAR_Task2_PySys_EntityBehavior_Scout2.log)
[Sim_STU2_SAR_Task2_PySys_MissionManager.log](https://github.com/user-attachments/files/17724051/Sim_STU2_SAR_Task2_PySys_MissionManager.log)
[Sim_STU2_SAR_Task2.log](https://github.com/user-attachments/files/17724052/Sim_STU2_SAR_Task2.log)

Here are all the logs as well as a csv file with the results of the lidar scan. By column, it's x, y, z, radii

**Comment by connorjak** [comment, marked as answer]
Fixed in 0.27.6

**Reply by connorjak** [reply]
The bug had all the previous scan results accumulating; fixed now.
